gateway/handler_success.go

```diff
@@ -352,3 +347,10 @@ func (s *SuccessHandler) ServeHTTPWithCache(w http.ResponseWriter, r *http.Reque
 
 	return inRes
 }
+
+func requestBodyGraphQL(r *http.Request) {
+	if gqlCtx := ctxGetGraphQLRequest(r); gqlCtx != nil {
```

Comment:
Why is that related to GraphQL requests?

Reply:
GraphQL middleware reads the request body and unmarshal it inside *gql.Request which is stored in the request context. @furkansenharputlu

Reply:
@furkansenharputlu there's something about that function that make it "mis-behave" for chunked body.
so the way to fix it is to comment these lines:
https://github.com/TykTechnologies/tyk/blob/master/gateway/reverse_proxy.go#L1604-L1606
https://github.com/TykTechnologies/tyk/blob/master/gateway/reverse_proxy.go#L1617-L1619

but it seems they mean something